chart/values.yaml

```diff
@@ -8,11 +8,13 @@ controller:
 
 # Command line flags for the agent
 agent:
-  hub_burst: 10 # int Allowed burst in requests/sec for accessing the hub from the agent (default 10)
```

Comment:
I do not know why GitHub is not highlighting this properly, but the change here is removing the redundant remark about default from the end of the line.